# == Camera Parameters == #

# Image dimensions
width: 640
height: 320

# Intrinsic
focal_x: 701.369863014
focal_y: 701.369863014
skew: 0.0 #-0.555483153496783
cx: 320
cy: 180
k1: 0
k2: 0
k3: 0
p1: 0
p2: 0

# Extrinsic
r_vec:
  - [0, -1,  0]
  - [0,  0, -1]
  - [1,  0,  0]

t_vec:
  - 0.003843059946351502
  - 0.6947744101849331
  - 1.8888207001689468

# BB Size coefficient

bb_size_coef: 1.3 # Coefficient from formula = 1.71740618